bletransportsample/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
-    id 'com.android.application'
-    id 'org.jetbrains.kotlin.android'
+    id('com.android.application').version("7.2.1")
+    id('org.jetbrains.kotlin.android').version("1.6.21")
```

Comment:
took it from https://github.com/bigman212/hw-transport-android-ble/commit/fafd6259784ce47f20501ea5337e94ce8fbfe3db